src/Compiler/Checking/NameResolution.fs

```diff
@@ -269,8 +269,8 @@ type Item =
         | Item.Event einfo -> einfo.DisplayNameCore 
         | Item.Property(_, pinfo :: _) -> pinfo.DisplayNameCore
         | Item.Property(nm, _) -> nm |> ConvertValLogicalNameToDisplayNameCore
-        | Item.MethodGroup(_, FSMeth(_, _, v, _) :: _, _) -> v.DisplayNameCore
-        | Item.MethodGroup(nm, _, _) -> nm |> ConvertValLogicalNameToDisplayNameCore
+        | Item.MethodGroup(methods = FSMeth(_, _, v, _) :: _) -> v.DisplayNameCore
+        | Item.MethodGroup(displayName= nm) -> nm |> ConvertValLogicalNameToDisplayNameCore
```

Comment:
Another nitpick but named access is formatted as `X(y = z)`.
See https://learn.microsoft.com/en-us/dotnet/fsharp/style-guide/formatting#formatting-named-arguments